client/login/magic-login/index.jsx

```diff
@@ -132,8 +133,20 @@ class MagicLogin extends Component {
 
 	renderLinks() {
 		const { isJetpackLogin, locale, showCheckYourEmail, translate } = this.props;
-
-		if ( showCheckYourEmail || this.props.query?.client_id ) {
+		const appPromo = (
+			<AppPromo
+				title={ translate( 'Stay logged in' ) }
+				campaign="emailed-login-link"
+				className="magic-link-app-promo"
+				subheader=""
```

Comment:
Is the `subheader` both controlling if the line should appear at all, as well as if the default text should be used? That can get confusing quickly.

`subheader=false` for not displaying might be clearer than empty string, or just having additional `hideSubheader` prop or similar.